scripts/dendrite_sytest.sh

```diff
@@ -61,12 +61,14 @@ fi
 
 # Check for new tests to be added to the test whitelist
 /src/show-expected-fail-tests.sh /logs/results.tap /src/sytest-whitelist \
-    /src/sytest-blacklist > /work/show_expected_fail_tests_output.txt || TEST_STATUS=$?
+    /src/sytest-blacklist | tee /work/show_expected_fail_tests_output.txt || TEST_STATUS=$?
```

Comment:
This would be very useful to have! Currently some sytest branches are `exit 1`ing because of this script and yet I have no idea why :S